packages/fxa-content-server/grunttasks/remarkable.js

```diff
@@ -12,15 +12,9 @@ module.exports = function (grunt) {
     // Normalize the filenames to use the locale name.
     // add the extension here, instead of using grunt-remarkable's
     // extension generator, to get locale names in uppercase
-    const lang = i18n.localeFrom(destFile.split('/')[0]);
 
-    if (destFile.includes('privacy')) {
-      return path.join(destPath, lang, 'privacy.html');
-    } else if (destFile.includes('tos')) {
-      return path.join(destPath, lang, 'terms.html');
-    } else {
-      return path.join(destPath, i18n.localeFrom(destFile) + '.html');
-    }
+    const lang = i18n.localeFrom(destFile.split('/')[0]);
+    return path.join(destPath, lang + '.html');
```

Comment:
for example, the path is something like `/server/templates/pages/dist/privacy/en_US.html`